### PR TITLE
Task/implement event messages

### PR DESCRIPTION
**Pull request by Jamie-Unosquare**
Event Messages Added
Simple functionality for adding and getting messages for General, Approval and Reject

### CODE REVIEW

File: v2/AdminCore.Common/Interfaces/IDashboardService.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using AdminCore.DTOs.Event;
 using System;
 using System.Collections.Generic;
+using AdminCore.DTOs.EventMessage;
```
Comment (nmcmahon-unosquare) [inline v2/AdminCore.Common/Interfaces/IDashboardService.cs:5]: Delete this

File: v2/AdminCore.Services/EventMessageService.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    public IList<EventMessageDto> GetAllEventMessagesForEvent(int eventId)
+    {
+      var eventMessages = DatabaseContext.EventMessageRepository.GetAsQueryable(x => x.EventId == eventId, null,
+                                                                                x => x.Employee)
+                                                                                .OrderByDescending(x => x.LastModified).ToList();
+      return _mapper.Map<IList<EventMessageDto>>(eventMessages);
+    }
+
+    public void CreateGeneralEventMessage(int eventId, string message)
+    {
```
Comment (nmcmahon-unosquare) [inline v2/AdminCore.Services/EventMessageService.cs:38]: this method is a chonker, refactor into 3 methods: buildEventMessage, addEventMessageToEvent, InsertEventMessageIntoDb (or something like this)

File: v2/AdminCore.WebApi.Tests/Controllers/DashboardControllerTest.cs [hunk 3]
```diff
@@ -9,6 +9,8 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using AdminCore.DTOs.EventMessage;
+using AdminCore.WebApi.Models.EventMessage;
```
Comment (nmcmahon-unosquare) [inline v2/AdminCore.WebApi.Tests/Controllers/DashboardControllerTest.cs:13]: revert file

File: v2/AdminCore.Common/Interfaces/IDashboardService.cs (hunk 1 above)
Comment (Jamie-Unosquare) [reply v2/AdminCore.Common/Interfaces/IDashboardService.cs:5]: EventMessage has been moved out of Dashboard directory so it needs to be imported

File: v2/AdminCore.WebApi.Tests/Controllers/DashboardControllerTest.cs (hunk 3 above)
Comment (Jamie-Unosquare) [reply v2/AdminCore.WebApi.Tests/Controllers/DashboardControllerTest.cs:13]: Same as above, these imports are needed as EventMessage has been moved

**Review by nmcmahon-unosquare — APPROVED** [review]

**Pull request merged**